grammars/level8-Additions.lark

```diff
@@ -1,6 +1,6 @@
 command:+= error_repeat_dep_8 | error_ifelse | error_ifpressed_missing_else | ifpressed ifpressed_elses? | ifs elses?-= error_invalid_space | ifelse
 
-repeat: _REPEAT (INT | var_access) _TIMES _SPACE? _EOL (_SPACE command) (_EOL _SPACE command)* _EOL? _END_BLOCK
+repeat.1: _REPEAT (INT | var_access) _TIMES _SPACE? _EOL (_SPACE command) (_EOL _SPACE command)* _EOL? _END_BLOCK
```

Comment:
I think adding this priority is fine, sometimes Hedy's grammar is difficult to disambiguate, and leveraging tricks like this to make it work is fine.